src/EditorFeatures/CSharpTest/CodeActions/IntroduceParameter/IntroduceParameterTests.cs

```diff
@@ -1990,5 +1990,40 @@ void localFunction(string s)
 
             await TestInRegularAndScriptAsync(code, expected, 0);
         }
+
+        [Fact]
+        public async Task TestIntroduceParameterOnAttributeArgument()
+        {
+            var code =
+@"
+public class C
+{
+    [Theory]
+    [InlineData([|""A""|])]
+    public void Test(string s)
+    {
+    }
+}
+";
+
+            await TestMissingInRegularAndScriptAsync(code);
+        }
+
+        [Fact]
+        public async Task TestIntroduceParameterOnArgument()
+        {
+            var code =
+@"
+public class C
+{
+    public void Test(int x)
+    {
+        Test([|5|]);
```

Comment:
Why are arguments not valid for introduce parameter?

Reply:
My mistake -- I only meant to filter out recursive arguments